Nintroller/Nintroller.csproj

```diff
@@ -58,15 +58,16 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Controllers/BalanceBoard.cs" />
-    <Compile Include="Controllers/ClassicController.cs" />
-    <Compile Include="Controllers/ClassicControllerPro.cs" />
-    <Compile Include="Controllers/Drums.cs" />
-    <Compile Include="Controllers/Guitar.cs" />
-    <Compile Include="Controllers/Nunchuk.cs" />
-    <Compile Include="Controllers/ProController.cs" />
-    <Compile Include="Controllers/Wiimote.cs" />
-    <Compile Include="Controllers/WiimotePlus.cs" />
+    <Compile Include="Controllers\BalanceBoard.cs" />
+    <Compile Include="Controllers\ClassicController.cs" />
+    <Compile Include="Controllers\ClassicControllerPro.cs" />
+    <Compile Include="Controllers\Drums.cs" />
+    <Compile Include="Controllers\Guitar.cs" />
+    <Compile Include="Controllers\Nunchuk.cs" />
+    <Compile Include="Controllers\ProController.cs" />
+    <Compile Include="Controllers\Turntable.cs" />
+    <Compile Include="Controllers\Wiimote.cs" />
+    <Compile Include="Controllers\WiimotePlus.cs" />
```

Comment:
These back slashes should be changed back to forward slashes for consistency.

Reply:
VS2022 wasn't able to find these files when the paths were forward slashes. Did you have to configure something in VS to allow forward slashes in paths?

Reply:
Ah, hm. I've been able to build just fine using VS Code and some tasks set up to invoke MSBuild manually, and it works fine. Good to know for the future though, I avoid using regular VS as much as possible because it doesn't run great on my PC lol

Reply:
Instead of changing them back to forward slashes, I changed the remaining forward slash to backslash
```xml
<Compile Include="Properties\AssemblyInfo.cs" />
```
This is actually consistent with WiitarThing.csproj, which also uses backslashes.

Reply:
Yeah, I went and checked and noticed that as well. Thanks for that!